app/controllers/users_controller.rb

```diff
@@ -13,6 +25,6 @@ def set_user
 
   # strong params
   def user_params
-    params.require(:user).permit(:name, :bio)
+    params.require(:user).permit(:name, :bio, :avatar)
```

Comment:
Nitpicking here. For most of our projects, we use Hound now to do our linting and one of the big things that comes up is lines being longer than 100 characters. So for methods like `permit` with multiple arguments, we usually write them as:
```
params.require(:user).permit(
  :name, 
  :bio, 
  :avatar
)
```

Your line isn't 100 characters yet, but could be in the future. So for continuity's sake, we like to format all of our methods with multiple arguments that way. Again, nitpicky and not a blocker by any means.